docs/changehistory/NextVersion.md

```diff
@@ -120,6 +120,10 @@ Several changes were made in the @bentley/ui-* packages.
 Some components in @bentley/ui-core were deprecated in favor of components in @itwinui-react.
 A few constructs were deprecated in @bentley/ui-core package with alternatives elsewhere.
 
+### @bentley/extension-cli
+
+Deprecated. It was unused and has been fully removed.
```

Comment:
Let's say something like, `The cli tool has been deprecated due to an impending change of Extensions and the Extension Service. Please continue to use the 2.x version if you still require publishing Extensions.`

Reply:
Updated doc comment